src/components/document-list/_macro.spec.js

```diff
@@ -4,472 +4,359 @@ import * as cheerio from 'cheerio';
+                                        iso: '2022-01-01',
+                                        short: '1 January 2022',
+                                    },
                                 },
                             },
-                        },
-                    ],
-                }),
-            );
-            expect($('time').attr('datetime')).toBe('2022-01-01');
+                        ],
+                    }),
+                );
+                const text = $('.ons-document-list__item-attribute > span').text().trim();
+                expect(text).toBe('Released:');
+            });
         });
+    });
 
-        it('has the correct `time` value', () => {
-            const $ = cheerio.load(
-                renderComponent('document-list', {
-                    documents: [
-                        {
-                            ...EXAMPLE_DOCUMENT_LIST_BASIC,
-                            metadata: {
-                                date: {
-                                    iso: '2022-01-01',
-                                    short: '1 January 2022',
+    describe('GIVEN: Params: showprefix', () => {
+        describe('WHEN: showprefix is set in the date metadata configuration for a document', () => {
+            test('THEN: applies the prefix class to the document item', () => {
+                const $ = cheerio.load(
+                    renderComponent('document-list', {
+                        documents: [
+                            {
+                                ...EXAMPLE_DOCUMENT_LIST_BASIC,
+                                metadata: {
+                                    date: {
+                                        showPrefix: true,
+                                        iso: '2022-01-01',
+                                        short: '1 January 2022',
+                                    },
                                 },
                             },
-                        },
-                    ],
-                }),
-            );
+                        ],
+                    }),
+                );
 
-            const time = $('.ons-document-list__item-attribute time').text().trim();
-            expect(time).toBe('1 January 2022');
+                expect($('.ons-document-list__item-attribute > span').hasClass('ons-u-fw-b')).toBe(true);
+            });
         });
     });
 
-    describe('mode: with all parameters', () => {
-        it('passes jest-axe checks', async () => {
-            const $ = cheerio.load(
-                renderComponent('document-list', {
-                    documents: [EXAMPLE_DOCUMENT_LIST_WITH_MULTIPLE],
-                }),
-            );
-
-            const results = await axe($.html());
-            expect(results).toHaveNoViolations();
-        });
-
-        it('has the correct document thumbnail class', async () => {
+    describe('GIVEN: Params: multiple', () => {
```

Comment:
Theres no multiple param, this test should come under `metadata`

Reply:
I am bit confused with these tests with multiple configurations under metadata param. As are testing each configuration separately, can we remove tests for multiple one and `EXAMPLE_DOCUMENT_LIST_WITH_MULTIPLE`?